docs/build/configure/_index.md

```diff
@@ -232,15 +232,26 @@ Find more information in the [processes documentation](/build/configure/processe
 ## Webhooks
 
 Webhooks allow you to trigger actions when certain types of data are sent from your machine to the cloud.
-For example, you can configure a webhook to send you a notification when your robot's sensor collects a new reading.
 
-Switch to **Raw JSON** mode to configure webhooks as follows:
+Viam offers different types of webhooks depending on the type of data you want to serve as a trigger.
+For example, you can use the `"part_data_ingested"` type to configure a webhook to send you a notification when your robot's sensor collects a new reading, or use the `"part_online"` type to notify you when your machine part comes online.
```

Comment:
IDK but I think we may have more that aren't surfaced. This looks good except I think whoever wrote this probably used "for example" on purpose because the action triggered does not have to be sending a notification, we are just documenting an easy/good use case

Reply:
Ok updated tried to keep flexibility. Checking in with naomi about other types

Reply:
also the configure instructions are meant to be flexibke with the notification function just suggested as an example

Reply:
Ooh you're totally right about the notification part, my bad. Your cloud function can indeed do pretty much anything at all, not just send a notification.
The list you added above LGTM. Now that we have the list, I think we can shorten this section for less redundancy; see suggestion for these two lines.